### DISCUSSION TITLE
How to properly set NavLink positioning?

### DISCUSSION THREAD

**Discussion by aliastopan** [microsoft/fluentui-blazor#1415] [Q&A] [answered]
Is there an options or a way to make NavLink (last or nth) item in NavMenu to be pushed all the way to the bottom of navigation bar?

Using FluentSpacer does not work unless the "width" is specified. But Width property only accept integer and cannot bet set to 'auto'

**Comment by vnbaaij** [comment, marked as answer]
No, we currently do not have any support for that. 

I would probably solve this by placing a NavLink at the bottom myself (so outside of the `FluentNavMenu`).
